docs/source/usage/workflows/addLaser.rst

```diff
@@ -7,18 +7,55 @@ Adding Laser
 
 There are several alternative ways of adding an incoming laser (or any source of electromagnetic field) to a PIConGPU simulation:
 
+#. selecting incident field profiles for respective boundaries in :ref:`incidentField.param <usage-params-core>`
 #. selecting a laser profile in :ref:`laser.param <usage-params-core>`
-#. selecting non-none incident field profiles for respective boundaries in :ref:`incidentField.param <usage-params-core>`
 #. using field or current background in :ref:`fieldBackground.param <usage-params-core>`
 
 These ways operate independently of one another, each has its features and limitations.
+Incident field- and laser profiles currently match one another.
+
+Incident Field
+""""""""""""""
+
+Incident field is an external field source producing a wave propagating inwards the simulation volume.
+The source is applied at a boundary of an axis-aligned box located inside the simulation area.
+The implementation is based on the total field/scattered field formulation described in detail :ref:`here <model-TFSF>`.
+A user sets offsets of each side of this box from global domain boundary in :ref:`incidentField.param <usage-params-core>`.
+Each offset must cover at least the field absorber thickness along the boundary so that the generating surface is located in the internal area.
+
+For each of the generation planes ``XMin, XMax, YMin, YMax, ZMin, ZMax`` (the latter two for 3d) a user sets incident profile to be applied.
+The configuration is done through parameter structures, depending on the profile type.
+Both profiles and parameter structures generally match their laser counterparts.
+The differences between matching incident field- and laser profiles are:
+
+# positioning of incident field is controlled for the generation plane and not via an internal member ``::initPlaneY``
+# incident field profiles do not have an extra time delay equal to :math:`initPlaneY * \Delta y / c` as lasers do (when needed, other parameters could be adjusted to accomodate for the delay)
+# default initial phase is chosen so that the laser starts smoothly at the generation plane (for laser it is always for plane :math:`y = 0`) 
```

Comment:
Please move the comment on how laser does it or remove it. I am sure we will forget to delete this comment when the whole old laser initialization is removed, and then it will be confusing.

Reply:
But this is the section comparing incident field to laser? So when we remove laser we remove this section at all

Reply:
Well, this is true. Hopefully we will remember to remove it. :sweat_smile: